### ISSUE TITLE
args.temperature has not been used in the loss?

### ISSUE THREAD

**Issue by pallashadow** [Microsoft/snca.pytorch#4]
The argument args.temperature=0.05 seems has never been used during training. Instead, an args.margin=0.0 is passed to the loss defined in lib/NCA.py.

**Comment by pallashadow** [comment]
it is in lemniscate object
